
UBC STAT 540 Project Proposal - January 2015
================================================
Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------

Project group members:
* Rashedul Islam
* Santina Lin
* Ka Ming Nip
* Eva Yap
* Beryl Zhuang

#### Background
Colorectal cancer (CRC) initiation and step-wise progression are associated with the accumulation of genetic and epigenetic alterations (Figure 1). The best-characterized epigenetic process is DNA methylation, which regulates gene transcription by covalent addition of a methyl group at the 5-carbon position of cytosine within cytosine-guanine dinucleotides (CpG). Aberrant DNA methylation can lead to malignancy by hypermethylation of CpG islands resulting in transcriptional silencing of tumour suppressor genes. CpG islands are sequences with high CpG fraction (>50%) located within gene promoters and methylation at these sites promotes association of methyl-binding proteins (MBPs) and subsequent recruitment of transcriptional repressors **(1,2)**.

Changes in DNA methylation is one of the early molecular events involved in CRC progression and many studies have identified these epigenetic abnormalities in precursor lesions such as aberrant crypt foci and adenomas **(3,4,5)**. Methylation aberrations can also mediate field defect, a phenomenon in which histologically normal tissues are predisposed to malignant transformation due to the presence of molecular alterations. For example, cancer-specific promoter methylation of the MGMT gene which encodes for a DNA repair protein was detected in normal mucosa adjacent to tumours and distant normal mucosa 10 cm away from tumours in CRC patients **(6)**. 

Therefore, comparison of DNA methylation patterns at the whole methylome level between normal tissues, precursor lesions, and colorectal tumours as well as comparison of tumours with paired normal tissues from CRC patients could provide further insights on the biology of CRC initiation and progression. Ultimately, these information could be translated into a clinically applicable tool which applies DNA methylation markers in early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC. 

#### Project Aims
1. To compare methylation patterns between normal tissues, colon adenomas, and colorectal tumour to identify aberrantly methylated CpG islands underlying CRC progression through the normal to adenoma and adenoma to tumour sequences.
2. To compare methylation patterns between paired normal tissues with tumours, which would help us characterize DNA methylation mediators of field defect in CRC. 

#### Dataset
For our group project, we will analyze DNA methylation array data from [Series GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"). This dataset is publicly available on the Gene Expression Omnibus (GEO) and is derived from four groups namely normal intestinal tissue from non-CRC patients (n = 17), normal intestinal tissue from CRC patients (n = 24), colon adenomas (n = 42), and colorectal tumours (n = 64).

Data for each sample consists of 

##### Technology used to generate the data
Illumina Infinium HumanMethylation450 (HM450) Bead- Chips (Illumina, San Diego, CA) were used to generate all data we will be using. The Infinium assay can detects methylation at a single base resolution. The length of methylation probe is 50 bp. The probes query a C to T conversion created by bisulfite treatment of unmethylated cytosines in the genome. The Infinium 450k methylation platform uses a combination of two distinct probe types, Infinium I and II **Figure 2a,b** **(4)**.

##### Data Filtering and Normalization
The data we collected were already normalized by previous authors **(1)**. At first they removed untrustworthy probes using a detection P value >.05. They used carried out normalization using a subset-quantile within array normalization (SWAN) which is a newly developed normalization method specifically designed for Illumina Infinium HM450 array data. SWAN also reduces technical variations within and between arrays. SWAN is a Bioconductor minfi package **(4)**. Batch effects across all array runs were assessed and corrected using the ComBat algorithm **(5)**. 

##### Data description
The data we will use has 3 columns and has few thousands rows. First column represent the probe name, second column for normalization value and third column for detection P-value. Each row shows single probe. [I will add an example table of data]

__(things to add?)__
- details about data files __(important)__
- the types of data (probes, samples,  how it's generated generated from the array) 

#### Proposed Methods
As revealed by the corresponding [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"), a total of 485,577 HumanMethylation450 probes were used in this study. Given our limited computational resources and time, we opt for downsizing the number of probes to those that only hybridize to CpG islands, which is widely studied and shown to be biologically relevant in cancer progression. 

*I think if we focus on CpG islands only, we should be able to downsize the probes enough. At least, that was what Alice's group did and What do you guys think?*

```
I have just verified. The GPL13534 data table has a column for 'Relation_to_UCSC_CpG_Island'.
- 150254 Island
-   24844 N_Shelf
-   62870 N_Shore
-   22300 S_Shelf
-   49197 S_Shore
- 176112 *blank*

~36% reduction if we remove blanks.
Relationship to Canonical CpG Island: Shores - 0-2 kb from CpG island; Shelves - 2-4 kb from CpG island.
~70% reduction if we focus on canonical CpGs only!!!
```

In the first phase of our project, we must ensure that we work with high-quality data. We would perform clustering analysis on data to see if clusters agree with their designated patient group. This step is very important because it can reveal sample-swaps. In addition, it also gives us a sense of the variability of our data. *Second step is to perform normalization of beta values, mention why and how? Or we can just leave it as perform data normalization. Third step is to perform clustering again post normalization and filtering.*

**TODO: we need to look into how those beta values were generated**

In order to do group comparison between the methylation data of the normal tissues, adenomas, and colorectal tumour, we will perform several statistical tests, such as ANOVA and t-test as covered recently in the lectures. We will also perform this statistical model for comparison between paired normal tissues with tumours in CRC patients. Depending on the remaining time and resource, we could also build prediction models based on our data to see if we could do prediction on the tumour progression based on the methylation data in early stages of colorectal cancer. 

#### References
<<<<<<< HEAD
1. Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2. Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3. Luo, Y. et al. Gastroenterol. 2014 Apr 23; 147(2):418–29.e8.
4. Chan, A. et al. Am. J. Pathol. 2002 Jan 24; 160(5):1823–30.
5. Muto, Y. et al. BMC Cancer. 2014 Jun 25; 14:466
6. Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
7. Pancione, M. et al. Pathology Res. Int. 2012 Jul 24; 2012:509348

#### Supplementary information
[Figure 1: Accumulation of genetic and epigenetic alterations during multistep progression of CRC. (5)](http://www.hindawi.com/journals/pri/2012/509348.fig.001b.jpg)
=======
1.  Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2.	Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3.	Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
4.	Maksimovic J et al. Genome Biol 2012; 13:R44.
5.	Johnson WE et al. Biostatistics 2007;8:118–127.

<<<<<<< Updated upstream
=======
>>>>>>> FETCH_HEAD
>>>>>>> Stashed changes
