
UBC STAT 540 Project Proposal - January 2015
================================================
Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------

Project group members:
* Rashedul Islam
* Santina Lin
* Ka Ming Nip
* Eva Yap
* Beryl Zhuang

#### Background
Colorectal cancer (CRC) initiation and step-wise progression are associated with the accumulation of genetic and epigenetic alterations (Figure 1). The best-characterized epigenetic process is DNA methylation, which regulates gene transcription by covalent addition of a methyl group at the 5-carbon position of cytosine within cytosine-guanine dinucleotides (CpG). Aberrant DNA methylation can lead to malignancy by hypermethylation of CpG islands resulting in transcriptional silencing of tumour suppressor genes. CpG islands are sequences with high CpG fraction (>50%) located within gene promoters and methylation at these sites promotes association of methyl-binding proteins (MBPs) and subsequent recruitment of transcriptional repressors **(1,2)**.

*Eva's job TBA - insert something about adenomas (what are they?)? something about field defect? How single-gene studies used to be performed and now that we're advancing to whole methylome studies which could provide more information for ...*

*blah blah blah...* Comparison of whole methylomes will not only provide insights on the biology of CRC initiation and progression, but it will also translate basic science research into a clinically applicable tool through the application of DNA methylation markers in early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC. 

#### Project Aims
1. To compare methylation patterns between normal tissues, adenomas, and colorectal tumour with the hope of identifying aberrantly methylated CpG islands underlying CRC progression through the normal to adenoma and adenoma to tumour sequences.
2. To compare methylation patterns between paired normal tissues with tumours, which would help us characterize DNA methylation mediators of field defect in CRC. 

*Should be sufficient to describe the dataset and goals? What do you think?*

__(things to add?)__
* other justifications

#### Dataset
For our group project, we will analyze DNA methylation array data from [Series GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"). This dataset is publicly available on the Gene Expression Omnibus (GEO) and is derived from normal intestinal tissue from non-CRC patients, normal intestinal tissue from CRC patients, adenomas, and colorectal tumours. 

__(things to add?)__
- the three groups (normal ,adenoma, CRC) :41 normal colon tissue, 42 colon adenomas,
and 64 cancers
- details about data files __(important)__
- the types of data (probes, samples,  how it's generated generated from the array) 

#### Proposed Methods
As revealed by the corresponding [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"), a total of 485,577 HumanMethylation450 probes were used in this study. Given our limited computational resources and time, we opt for downsizing the number of probes to those that only hybridize to CpG islands, which is widely studied and shown to be biologically relevant in cancer progression. 

*I think if we focus on CpG islands only, we should be able to downsize the probes enough. At least, that was what Alice's group did and What do you guys think?*

**TODO: We will have to overlap the probe coodinates with the liftover track for cpg islands and  see.**

In the first phase of our project, we must ensure that we work with high-quality data. We would perform clustering analysis on data to see if clusters agree with their designated patient group. This step is very important because it can reveal sample-swaps. In addition, it also gives us a sense of the variability of our data. *Second step is to perform normalization of beta values, mention why and how? Or we can just leave it as perform data normalization. Third step is to perform clustering again post normalization and filtering.*

**TODO: we need to look into how those beta values were generated**

In order to do group comparison between the methylation data of the normal tissues, adenomas, and colorectal tumour, we will perform several statistical tests, such as ANOVA and t-test as covered recently in the lectures. We will also perform this statistical model for comparison between paired normal tissues with tumours in CRC patients. Depending on the remaining time and resource, we could also build prediction models based on our data to see if we could do prediction on the tumour progression based on the methylation data in early stages of colorectal cancer. 

#### References
1.  Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2.	Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3.	Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.

