
UBC STAT 540 Project Proposal - January 2015
================================================
Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------

Project group members:
* Rashedul Islam
* Santina Lin
* Ka Ming Nip
* Eva Yap
* Beryl Zhuang

#### Background
Colorectal cancer (CRC) initiation and step-wise progression are associated with the accumulation of genetic and epigenetic alterations **(Figure 1)**. The best-characterized epigenetic process is DNA methylation, which regulates gene transcription by covalent addition of a methyl group at the 5-carbon position of cytosine within cytosine-guanine dinucleotides (CpG). Aberrant DNA methylation can lead to malignancy by hypermethylation of CpG islands resulting in transcriptional silencing of tumour suppressor genes. CpG islands are sequences with high CpG fraction (>50%) located within gene promoters and methylation at these sites promotes association of methyl-binding proteins (MBPs) and subsequent recruitment of transcriptional repressors **(1,2)**.

Changes in DNA methylation is one of the early molecular events involved in CRC progression and many studies have identified these epigenetic abnormalities in precursor lesions such as aberrant crypt foci and adenomas **(3,4,5)**. Methylation aberrations can also mediate field defect, a phenomenon in which histologically normal tissues are predisposed to malignant transformation due to the presence of molecular alterations. For example, cancer-specific promoter methylation of the O<sup>6</sup>-methylguanine DNA methyltransferase (*MGMT*) gene which encodes for a DNA repair protein was detected in normal intestinal tissues adjacent to tumours and distant normal tissues 10 cm away from tumours in CRC patients **(6)**. 

Therefore, comparison of DNA methylation patterns at the whole methylome level between normal tissues, precursor lesions, and colorectal tumours as well as comparison of tumours with paired normal tissues from CRC patients could provide further insights on the biology of CRC initiation and progression. Ultimately, these information could be translated into a clinically applicable tool which applies DNA methylation markers in early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC. 

#### Project Aims
1. To compare methylation patterns between normal tissues, colon adenomas, and colorectal tumours to identify aberrantly methylated CpG islands underlying CRC progression through the normal to adenoma and adenoma to tumour sequences.
2. To compare methylation patterns between normal tissues from healthy patients and normal tissues from CRC patients to characterize DNA methylation mediators of field defect. 

#### Dataset
For our group project, we will analyze DNA methylation array data from [Series GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"). This dataset is publicly available on the Gene Expression Omnibus (GEO) and is derived from four sample groups namely normal intestinal tissues from four different patient groups: 

| Full name |  shorthand| size |
| ------------- |:-------------:| -----:|
| Genomic DNA from normal individuals      | normal | 17 |
| Genomic DNA of normal colon sample from CRC patients  | CRCnormal  |    24  |
| Genomic DNA from adenoma patients |    adenoma   |   42  |
| Genomic DNA from CRC patients | CRC |   64  |

The dataset we chose to analyze has been normalized by Luo *et al*. (2014). As mentioned under Supplementary Methods of the paper **(3)**, untrustworthy probes using a detection P value >.05 were removed. The detection P-values estimate the difference of the signal intensities of CpG site compared with negative control probes embedded in the assay **(7)**. This step is followed by normalization for technical variations with the SWAN package from BioConductor. Lastly, correction for batch effects was performed using the ComBat algorithm **(8,9)**. The datasets we will use have three columns. First column represents the probe name, second column for average beta value and third column for detection P-value. Each row shows result from a single probe. Here is an example of the 3 column dataset:

[Sample GSM1183439](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1183439) showing the first 4 rows:

ID_REF	| VALUE |	Detection Pval
------------- | -------------|------------- |
cg00000029	|0.3459326|	0.00
cg00000108  |0.9509044|	0.00
cg00000109	|0.8988764|	0.00
cg00000165	|0.248589|	0.00
...|...|...

#### Proposed Methods
As revealed by the corresponding [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"), a total of 485,577 HumanMethylation450 probes were used in this study. Given our limited computational resources and time, we opt for downsizing the number of probes to those (150,254) that hybridize to CpG islands, which are widely studied and shown to be biologically relevant in cancer progression as discussed earlier. We also plan to filter out probes that contain single nucleotide polymorphisms (SNPs), that hybridize to non-CpG sites, that contain a SNP within 10 bp from the query sites and that are associated with chromosome X. 

In the first phase of our project, we want to do a sanity check on the on the quality of the data. We will perform unsupervised hierarchical [clustering analysis](http://www.statmethods.net/advstats/cluster.html) on our data to see if samples from the same patient groups are clustered together. If the majority of the samples are correctly clustered in the same patient groups, the analysis could suggest that the few ones that are incorrectly clustered might be sample-swaps or outliers. If our clustering analysis fails to correctly group the samples from the same groups together, it would suggest that the beta values of a large numbers of our probes do not significnatly differ between groups. 

We will then perform normalization on our data and then clustering analysis to see if the results differ. To further examine the data prior to the group comparison analysis, we will also examine covariance across the samples, assuming that samples in the same patient groups have strong co-variance. 

In order to do group comparison between the methylation data of the normal tissues, adenomas, and colorectal tumour, we will perform several statistical tests, including ANOVA and limma as covered in the recent lectures. To examine the different stages of CRC progression, we will do two group comparisons for the following pairs:
- CRCnormal and normal : to see, if any, regions in the genome that are differently methylated. This could reveal whether patients with CRC have different epigenomics in their normal tissues. 
- normal and adenoma : to see if any regions become differently methylated in the progression from normal tissue to development of adenoma.  
- adenoma and CRC : to see, if any, which probes indicate different methylation levels between the two groups. This information could reveal possible epigenomics changes as adenoma progresses to CRC. 

We will then find the genes that are associated with our top-hit probes and then do functional enrichement analysis on this list of genes, as part of our Aim #1. By examining the pathways these genes are part of, it could help us evaluate our analysis results and whether epigenomics play an important role in CRC.

#### References
1. Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2. Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3. Luo, Y. et al. Gastroenterol. 2014 Apr 23; 147(2):418–29.e8.
4. Chan, A. et al. Am. J. Pathol. 2002 Jan 24; 160(5):1823–30.
5. Muto, Y. et al. BMC Cancer. 2014 Jun 25; 14:466
6. Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
7. Hinoue T et al. Genome Res 2012;22:271–282.
8. Maksimovic J et al. Genome Biol 2012; 13:R44.
9. Johnson WE et al. Biostatistics 2007;8:118–127.
10. Pancione, M. et al. Pathology Res. Int. 2012 Jul 24; 2012:509348

#### Supplementary information
![Figure 1: Accumulation of genetic and epigenetic alterations during multistep progression of CRC. (10)](http://www.hindawi.com/journals/pri/2012/509348.fig.001b.jpg)

Figure 1: Accumulation of genetic and epigenetic alterations during multistep progression of CRC. __(10)__

