
UBC STAT 540 Project Proposal - January 2015
================================================
Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------

Project group members:
* Rashedul Islam
* Santina Lin
* Ka Ming Nip
* Eva Yap
* Beryl Zhuang

#### Background
Colorectal cancer (CRC) initiation and step-wise progression are associated with the accumulation of genetic and epigenetic alterations **(Figure 1)**. The best-characterized epigenetic process is DNA methylation, which regulates gene transcription by covalent addition of a methyl group at the 5-carbon position of cytosine within cytosine-guanine dinucleotides (CpG). Aberrant DNA methylation can lead to malignancy by hypermethylation of CpG islands resulting in transcriptional silencing of tumour suppressor genes. CpG islands are sequences with high CpG fraction (>50%) located within gene promoters and methylation at these sites promotes association of methyl-binding proteins (MBPs) and subsequent recruitment of transcriptional repressors **(1,2)**.

Changes in DNA methylation is one of the early molecular events involved in CRC progression and many studies have identified these epigenetic abnormalities in precursor lesions such as aberrant crypt foci and adenomas **(3,4,5)**. Methylation aberrations can also mediate field defect, a phenomenon in which histologically normal tissues are predisposed to malignant transformation due to the presence of molecular alterations. For example, cancer-specific promoter methylation of the O<sup>6</sup>-methylguanine DNA methyltransferase (*MGMT*) gene which encodes for a DNA repair protein was detected in normal intestinal tissues adjacent to tumours and distant normal tissues 10 cm away from tumours in CRC patients **(6)**. 

Therefore, comparison of DNA methylation patterns at the whole methylome level between normal tissues, precursor lesions, and colorectal tumours as well as comparison of tumours with paired normal tissues from CRC patients could provide further insights on the biology of CRC initiation and progression. Ultimately, these information could be translated into a clinically applicable tool which applies DNA methylation markers in early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC. 

#### Project Aims
1. To compare methylation patterns between normal tissues, colon adenomas, and colorectal tumours to identify aberrantly methylated CpG islands underlying CRC progression through the normal to adenoma and adenoma to tumour sequences.
2. To compare methylation patterns between tumours with paired normal tissues from CRC patients to characterize DNA methylation mediators of field defect. 

#### Dataset
For our group project, we will analyze DNA methylation array data from [Series GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"). This dataset is publicly available on the Gene Expression Omnibus (GEO) and is derived from four sample groups namely normal intestinal tissues from non-CRC patients (n = 17), normal intestinal tissues from CRC patients (n = 24), colon adenomas (n = 42), and colorectal tumours (n = 64).

Illumina Infinium HumanMethylation450 (HM450) Bead- Chips (Illumina, San Diego, CA) were used to generate all data we will be using. The length of methylation probe is 50 bp. The data we collected were already normalized by previous authors **(3)**. At first they removed untrustworthy probes using a detection P value >.05. The detection P-values estimate the difference of the signal intensities of CpG site compared with negative control probes embedded in the assay **(7)**. The data we use for analysis were normalized for technical variations with the SWAN package from BioConducted and analyzed for batch effects with COMBAT **(8,9)**. The datasets we will use has 3 columns. First column represent the probe name, second column for normalization value and third column for detection P-value. Each row shows single probe. Here is an example of the 3 column dataset:

An example of [Sample GSM1183439](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1183439) showing the first 4 rows:

ID_REF	| VALUE |	Detection Pval
------------- | -------------|------------- |
cg00000029	|0.3459326|	0.00
cg00000108  |0.9509044|	0.00
cg00000109	|0.8988764|	0.00
cg00000165	|0.248589|	0.00
...|...|...

#### Proposed Methods
As revealed by the corresponding [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"), a total of 485,577 HumanMethylation450 probes were used in this study. Given our limited computational resources and time, we opt for downsizing the number of probes to those (150,254) that hybridize to CpG islands, which is widely studied and shown to be biologically relevant in cancer progression.

In the first phase of our project, we must ensure that we work with high-quality data. We would perform clustering analysis on data to see if clusters agree with their designated patient group. This step is very important because it can reveal sample-swaps. In addition, it also gives us a sense of the variability of our data.

In order to do group comparison between the methylation data of the normal tissues, adenomas, and colorectal tumour, we will perform several statistical tests, such as ANOVA and t-test as covered recently in the lectures. We will also perform this statistical model for comparison between tumours with paired normal tissues in CRC patients. Depending on the remaining time and resource, we could also build prediction models based on our data to see if we could predict tumour progression based on the methylation data in early stages of CRC. 

#### References
1. Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2. Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3. Luo, Y. et al. Gastroenterol. 2014 Apr 23; 147(2):418–29.e8.
4. Chan, A. et al. Am. J. Pathol. 2002 Jan 24; 160(5):1823–30.
5. Muto, Y. et al. BMC Cancer. 2014 Jun 25; 14:466
6. Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
7. Hinoue T et al. Genome Res 2012;22:271–282.
8. Maksimovic J et al. Genome Biol 2012; 13:R44.
9. Johnson WE et al. Biostatistics 2007;8:118–127.
10. Pancione, M. et al. Pathology Res. Int. 2012 Jul 24; 2012:509348

#### Supplementary information
![Figure 1: Accumulation of genetic and epigenetic alterations during multistep progression of CRC. (5)](http://www.hindawi.com/journals/pri/2012/509348.fig.001b.jpg)

Figure 1: Accumulation of genetic and epigenetic alterations during multistep progression of CRC. (5)

