## UBC STAT 540 Project Proposal - January 2015
================================================
### Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------
#### Deadline: Feb 2, 2015

#### Group Members

Member	| Graduate Program |	Lab Group | Interest/Expertise |
------------- | -------------|------------- |------------- |
Rashedul Islam @Rashedul	|Bioinformatics Training Program| Dr. Martin Hirst (CHiBi) | Bioinformatics |
Santina Lin @Santina  |Bioinformatics Training Program| Dr. Steven Jones (BCGSC) | Data visualization, data analysis, data mining  |	
Ka Ming Nip @kmnip	|Bioinformatics| Dr. Inanc Birol (BCGSC) | Data Visualization, RNA-seq Assembly, Structural Variation Analyses, Virology, Pathogenomics|
Eva Yap	@evayap|Experimental Medicine|	Dr. Aly Karsan (BCGSC) | Biochemistry |
Beryl Zhuang @BerylZhuang	|Bioinformatics Training Program| Dr. William Hsiao (BCCDC) | Genetics, Application of Bioninformatics tools |	
- CHiBi = Centre for High-Throughput Biology
- BCGSC = Canada's Michael Smith Genome Sciences Centre		
- BCCDC = BC Centre for Disease Control


#### Project proposal
Colorectal cancer (CRC) step-wise progression is associated with the accumulation of aberrant DNA methylation. This includes hypermethylation of CpG islands which are sequences with high CpG fraction (>50%) located within gene promoters resulting in transcriptional silencing of tumour suppressor genes **(1,2)**. Changes in DNA methylation is one of the early molecular events involved in CRC progression and many studies have identified these epigenetic abnormalities in precursor lesions such as aberrant crypt foci and adenomas **(3,4,5)**. Methylation aberrations can also mediate field defect, a phenomenon in which histologically normal tissues are predisposed to malignant transformation due to the presence of molecular alterations **(6)**. In this project, we aim to compare DNA methylation patterns (specifically hypermethylation of CpG islands) between normal tissues, colon adenomas, and colorectal tumours as well as comparison of tumours with paired normal tissues from CRC patients. To achieve this, we will analyze DNA methylation array data from [GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534), which is publicly available on the Gene Expression Omnibus (GEO). This dataset consists of four sample groups namely normal intestinal tissues from non-CRC patients (n = 17), normal intestinal tissues from CRC patients (n = 24), colon adenomas (n = 42), and colorectal tumours (n = 64). Identifying specific DNA methylation markers underlying CRC progression could be translated into a clinically applicable tool to improve early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC.

#### References
1. Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2. Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3. Luo, Y. et al. Gastroenterol. 2014 Apr 23; 147(2):418–29.e8.
4. Chan, A. et al. Am. J. Pathol. 2002 Jan 24; 160(5):1823–30.
5. Muto, Y. et al. BMC Cancer. 2014 Jun 25; 14:466.
6. Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
