## UBC STAT 540 Project Proposal - January 2015
================================================
### Identifying aberrant methylation patterns underlying colorectal cancer progression
----------------
#### Deadline: Feb 2, 2015

#### Project group members

Members	| Graduate Program |	Lab Group | Interest/Expertise |
------------- | -------------|------------- |------------- |
Rashedul Islam	|Bioinformatics Training Program| Awesome lab | Being awesome |
Santina Lin  |Bioinformatics Training Program| Awesome lab | Being awesome |
Ka Ming Nip	|Bioinformatics Training Program| Awesome lab | Being awesome |
Eva Yap	|Experimental Medicine|	Karsan Lab | Biochemistry |
Beryl Zhuang	|Bioinformatics Training Program| Awesome lab | Being awesome |

#### Project proposal
Colorectal cancer (CRC) initiation and step-wise progression are associated with the accumulation of genetic and epigenetic alterations. The best-characterized epigenetic process is DNA methylation, which regulates gene transcription by covalent addition of a methyl group at the 5-carbon position of cytosine within cytosine-guanine dinucleotides (CpG). Aberrant DNA methylation can lead to malignancy by hypermethylation of CpG islands which are sequences with high CpG fraction (>50%) located within gene promoters and this results in transcriptional silencing of tumour suppressor genes **(1,2)**. Changes in DNA methylation is one of the early molecular events involved in CRC progression and many studies have identified these epigenetic abnormalities in precursor lesions such as aberrant crypt foci and adenomas **(3,4,5)**. Methylation aberrations can also mediate field defect, a phenomenon in which histologically normal tissues are predisposed to malignant transformation due to the presence of molecular alterations. For example, cancer-specific promoter methylation of the O<sup>6</sup>-methylguanine DNA methyltransferase (*MGMT*) gene which encodes for a DNA repair protein was detected in normal intestinal tissues adjacent to tumours and distant normal tissues 10 cm away from tumours in CRC patients **(6)**. 

For our group project, we aim to compare DNA methylation patterns at the whole methylome level between normal tissues, colon adenomas, and colorectal tumours as well as comparison of tumours with paired normal tissues from CRC patients. To achieve this, we will analyze DNA methylation array data from [Series GSE48684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48684) generated by the Illumina HumanMethylation450 BeadChip platform [GEO Platform GPL13534](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13534 "Platform GPL13534"), which is publicly available on the Gene Expression Omnibus (GEO). This dataset consists of four sample groups namely normal intestinal tissues from non-CRC patients (n = 17), normal intestinal tissues from CRC patients (n = 24), colon adenomas (n = 42), and colorectal tumours (n = 64). We hope to identify specific DNA methylation markers underlying CRC progression which could be translated into a clinically applicable tool to improve early detection, risk assessment, disease monitoring, and therapeutic intervention of CRC.

#### References
1. Li, J. et al. ISRN Gastroenterol. 2014 Mar 6; 2014:464015.
2. Vaiopoulos, A. G. et al. Biochim. Biophys. Acta. 2014 Feb 20; 1842(7):971–80.
3. Luo, Y. et al. Gastroenterol. 2014 Apr 23; 147(2):418–29.e8.
4. Chan, A. et al. Am. J. Pathol. 2002 Jan 24; 160(5):1823–30.
5. Muto, Y. et al. BMC Cancer. 2014 Jun 25; 14:466.
6. Shen, L. et al. J. Natl. Cancer Inst. 2005 Sept 21; 97(18):1330–8.
