---
title: "DAVID"
author: "Rashedul"
date: "Sunday, April 05, 2015"
output:
  html_document:
    keep_md: yes
---

[DAVID](http://david.abcc.ncifcrf.gov/) 

**Input:** One column gene id without header row and ids are case sensitive. Example, 
 
```
1000_at
1001_at
1002_at
```

**Limit of gene Number:** Preferable for 3000 genes and returns results withing one minute. However, DAVID can take more than 3000 genes.

**Analysis:** We will do the most popular results like "Funtional annotation chart" and "Funtional annotation clustering".

**Background:** Enrichment analysis compares the annotation of a gene to the population background genes. The default background is for the genome-wide enrichment analysis studies. Available backgrounds are  Affymetrix chip and Illumina chip backgrounds as well as suer defined background. Affymetrix chip and Illumina chip backgrounds is a better choice for a gene list derived from Affymetrix microarray or Illumina studies, respectively.

**Method:** Used following code to extract ENSEMBL GENE ID from the probes. Then in DAVID analysis page [here](http://david.abcc.ncifcrf.gov/tools.jsp) uploaded the Ensebbl gene id (Step 1). Selected "ENSEMBL_GENE_ID" (Step 2) and selected "Gene list" (Step 3) and finally submit (Step 4).

```{r}
#Get ENTREX_GENE_ID from 'IlluminaHumanMethylation450kENTREZID'

x <- IlluminaHumanMethylation450kENSEMBL
# Get the entrez gene IDs that are mapped to an Ensembl ID
mapped_genes <- mappedkeys(x)
# Convert to a list
xx <- as.list(x[mapped_genes])
if(length(xx) > 0) {
  # Get the Ensembl gene IDs for the first five genes
  xx[1:5]
  # Get the first one
  #xx[[1]]
}
```

**Results:** 

