# Hierarchical Clustering Analysis
Santina  
Monday, March 16, 2015  

Beryl and Ka Ming has written the scripts for the rest of the group to download and filter raw data and raw meta data. Here I will try some clustering analysis to test the water... or look at the data at a high level.

The goal is to see if there are distinguishable features between different patient groups that clustering methods that come with R packages can pick out and correctly group the samples into their corresponding groups. 

This report is purely for internal communication. 

## Load the data

First we'll load the data:


```r
# run the script that gets the raw data and filter the raw data
# this step would take a while 
start <- proc.time()
source('../rscripts/get_data_and_filter.R')
(time <- proc.time() - start)

# run the script that take the metadata on our repo and process/filter it 
# reads in "GSE48684_metadata_raw.Rdata""
source('../rscripts/process_metadata.R')
load("../data/metadata.Rdata")
mD <- metadata
```

I am having trouble running the first script... Probably because (as Ka Ming suggested) that I have corrupted temp file and the script is using that. Anyway, I got the file from Beryl so I'll just use that : 


```r
load('../data/GSE48684_raw_filtered.Rdata')
data <- raw_data_filter
load('../data/metadata.Rdata')
meta <- metadata
head(data)
```

```
##            GSM1183439 GSM1183440 GSM1183441 GSM1183442 GSM1183443
## cg00050873 0.87153040 0.89495460  0.6093750 0.87842130 0.89606860
## cg00063477 0.89772560 0.93803160  0.4852941 0.94228640 0.95133270
## cg00121626 0.70276340 0.44292580  0.5263158 0.54055630 0.54761520
## cg00212031 0.06251355 0.03182913  0.5037221 0.06570613 0.01812475
## cg00213748 0.77986580 0.86277300  0.3082902 0.77699040 0.83620970
## cg00214611 0.02722982 0.02486223  0.4694960 0.03288129 0.01453940
##            GSM1183444 GSM1183445 GSM1183446 GSM1183447 GSM1183448
## cg00050873 0.86411680  0.5121107  0.6389892 0.70725050  0.5295275
## cg00063477 0.92998790  0.5389049  0.5275311 0.92888770  0.5908558
## cg00121626 0.43066520  0.2141328  0.4771574 0.83613020  0.5928074
## cg00212031 0.07941718  0.5557692  0.3333333 0.80034610  0.3005952
## cg00213748 0.80006250  0.2034549  0.2849003 0.80853740  0.3333333
## cg00214611 0.02377164  0.4161850  0.3505976 0.02797866  0.3442623
##            GSM1183449 GSM1183450 GSM1183451 GSM1183452 GSM1183453
## cg00050873  0.2973568 0.70984630  0.3869048  0.6153846 0.86182330
## cg00063477  0.5849515 0.93012270  0.4814443  0.5920398 0.93242850
## cg00121626  0.5298869 0.87283840  0.6103647  0.4634921 0.56785230
## cg00212031  0.5271868 0.03598251  0.2160714  0.3605442 0.03748895
## cg00213748  0.3973684 0.45862330  0.3421053  0.3751962 0.85509880
## cg00214611  0.4640884 0.03857968  0.3846154  0.3744557 0.03749888
##            GSM1183454 GSM1183455 GSM1183456 GSM1183457 GSM1183458
## cg00050873  0.8590089  0.4692654  0.4087049  0.3631436 0.56435640
## cg00063477  0.9196702  0.7179126  0.5588526  0.5238096 0.93133070
## cg00121626  0.3868635  0.4238329  0.5593804  0.4598765 0.82784880
## cg00212031  0.2068620  0.5717345  0.4363839  0.5110024 0.72666670
## cg00213748  0.8282855  0.6888217  0.4367161  0.1828154 0.33466940
## cg00214611  0.0404234  0.4250000  0.4284369  0.3012048 0.02995745
##            GSM1183459 GSM1183460 GSM1183461 GSM1183462 GSM1183463
## cg00050873 0.82299510  0.3824651  0.3423680  0.5623633  0.3039443
## cg00063477 0.94567850  0.6595136  0.3615108  0.3333333  0.3442623
## cg00121626 0.72420730  0.5652174  0.4764398  0.4595959  0.3866091
## cg00212031 0.10261210  0.3793911  0.3485342  0.2247191  0.2531646
## cg00213748 0.90155040  0.4753363  0.2541322  0.2568182  0.3966245
## cg00214611 0.03672594  0.4328358  0.3002915  0.6233522  0.3333333
##            GSM1183464 GSM1183465 GSM1183466 GSM1183467 GSM1183468
## cg00050873 0.73454810 0.85353420 0.81121810 0.92154580 0.90685730
## cg00063477 0.95604570 0.94717890 0.91421810 0.94526280 0.94857950
## cg00121626 0.60795860 0.59269700 0.56529960 0.49371620 0.56503640
## cg00212031 0.18048430 0.16676570 0.06666667 0.04546738 0.03109166
## cg00213748 0.68652480 0.79035850 0.68508820 0.74157300 0.76825130
## cg00214611 0.03304917 0.02896837 0.08635347 0.04772503 0.04377390
##            GSM1183469 GSM1183470 GSM1183471 GSM1183472 GSM1183473
## cg00050873  0.5355330  0.4969697 0.90055360 0.89781310 0.92086960
## cg00063477  0.3430962  0.4666667 0.93374040 0.93762370 0.93582060
## cg00121626  0.1416431  0.1952862 0.53654740 0.54731990 0.44727490
## cg00212031  0.4097561  0.3664773 0.07481225 0.04508009 0.01831101
## cg00213748  0.5123042  0.2635659 0.73453790 0.69360000 0.80093460
## cg00214611  0.1361502  0.4709302 0.04622622 0.03019370 0.02881483
##            GSM1183474 GSM1183475 GSM1183476 GSM1183477 GSM1183478
## cg00050873  0.4170854  0.4747899 0.92614010  0.4409938  0.4527629
## cg00063477  0.4519906  0.4601770 0.95941530  0.4875519  0.4571949
## cg00121626  0.1468927  0.2627551 0.49018140  0.4310954  0.2918455
## cg00212031  0.3129973  0.4973822 0.07328605  0.3897912  0.3750000
## cg00213748  0.3447368  0.4355231 0.78783380  0.4323432  0.3530752
## cg00214611  0.4739454  0.4836363 0.03746098  0.4265060  0.4171598
##            GSM1183479 GSM1183480 GSM1183481 GSM1183482 GSM1183483
## cg00050873  0.4153578 0.93424870 0.91587300 0.93807380  0.5286260
## cg00063477  0.3669065 0.94281210 0.94299970 0.94542500  0.5042735
## cg00121626         NA 0.60815500 0.40641330 0.48376730  0.2218310
## cg00212031  0.4712153 0.18326490 0.06144121 0.08656330  0.4176904
## cg00213748  0.1732523 0.79128860 0.77806560 0.77092510  0.3670520
## cg00214611  0.4845361 0.03657169 0.03117807 0.05721271  0.3197115
##            GSM1183484 GSM1183485 GSM1183486 GSM1183487 GSM1183488
## cg00050873 0.92864580 0.92826730 0.92077630 0.90449330 0.91522020
## cg00063477 0.94215980 0.94377240 0.95005230 0.94291750 0.94856870
## cg00121626 0.70471570 0.54972830 0.59913580 0.67375300 0.60361580
## cg00212031 0.09431681 0.10583280 0.06311064 0.04783063 0.08472776
## cg00213748 0.78136530 0.77607110 0.80375000 0.83134920 0.83879260
## cg00214611 0.07723089 0.03979639 0.04781514 0.03549952 0.04047569
##            GSM1183489 GSM1183490 GSM1183491 GSM1183492 GSM1183493
## cg00050873  0.8976546  0.5000000  0.3491525  0.5352423  0.4477318
## cg00063477  0.9546450  0.3685345  0.4543478  0.5308642  0.4063492
## cg00121626  0.6352769  0.3176101  0.2187500  0.2318501  0.5224490
## cg00212031  0.0810676  0.4206774  0.5774648  0.4288889  0.3683084
## cg00213748  0.7937132  0.3017903  0.3526971  0.3916667  0.2817338
## cg00214611  0.0548639  0.4070981  0.3772242  0.3737864  0.4749340
##            GSM1183494 GSM1183495 GSM1183496 GSM1183497 GSM1183498
## cg00050873 0.92096100  0.6989915 0.91579760  0.4326425  0.4812834
## cg00063477 0.94968430  0.5000000 0.96721830  0.4580153  0.5153509
## cg00121626 0.58231530  0.3760331 0.74062710  0.3378839  0.4312169
## cg00212031 0.04249292  0.4352617 0.10102740  0.1529412  0.3562232
## cg00213748 0.80875000  0.4240506 0.68392850  0.2960289  0.4084159
## cg00214611 0.03627907  0.3137755 0.04957102  0.3544304  0.4970760
##            GSM1183499 GSM1183500 GSM1183501 GSM1183502 GSM1183503
## cg00050873  0.4270152  0.5098591  0.3617977  0.5911765  0.3702703
## cg00063477  0.3483146  0.5841584  0.4888508  0.4809322  0.4738956
## cg00121626  0.2625000  0.2766571  0.4048781  0.3694915  0.2100656
## cg00212031  0.4148681  0.3472222  0.5013967  0.2365270  0.3790850
## cg00213748  0.4738806  0.3313433  0.2579787  0.4415275  0.4099526
## cg00214611  0.4184783  0.4866530  0.5083682  0.5173501  0.5365297
##            GSM1183504 GSM1183505 GSM1183506 GSM1183507 GSM1183508
## cg00050873  0.4193548  0.4458716  0.4619205 0.84690860  0.4000000
## cg00063477  0.3898734  0.4378819  0.5571429 0.95201210  0.4790997
## cg00121626  0.1439394  0.3419913  0.3848797 0.32982650  0.3670360
## cg00212031  0.4172494  0.4069767  0.3510848 0.86133010  0.4642263
## cg00213748  0.4153226  0.2045455  0.3581213 0.94507930  0.4015296
## cg00214611  0.3613445  0.2436261  0.3079777 0.02505718  0.3027027
##            GSM1183509 GSM1183510 GSM1183511 GSM1183512 GSM1183513
## cg00050873  0.4413646 0.82914440  0.3543544  0.4313726  0.3457447
## cg00063477  0.4787018 0.94488970  0.5164835  0.5654677  0.5240384
## cg00121626         NA 0.85880940  0.3827493  0.4698163  0.3957597
## cg00212031  0.4819588 0.02841430  0.2416107  0.4110672  0.4299803
## cg00213748  0.2926045 0.53196930  0.3759791  0.2890995  0.4736842
## cg00214611  0.3210702 0.02251965  0.5181818  0.3733583  0.4054622
##            GSM1183514 GSM1183515 GSM1183516 GSM1183517  GSM1183518
## cg00050873  0.3713163  0.9101942 0.87450500  0.3715711 0.899630800
## cg00063477  0.4200820  0.7428572 0.92430440  0.4990138 0.925167400
## cg00121626  0.4860759  0.5487013 0.76233570  0.3647541 0.680847000
## cg00212031  0.2752613  0.1439153 0.46220470  0.3069054 0.075096280
## cg00213748  0.4074074  0.4606481 0.74082070  0.1202346 0.649762300
## cg00214611  0.3719807  0.1812500 0.04021318  0.4698795 0.005164622
##            GSM1183519 GSM1183520 GSM1183521 GSM1183522 GSM1183523
## cg00050873  0.8839869  0.4259681  0.9239234 0.85698800 0.91863070
## cg00063477  0.8998944  0.5009901  0.7777778 0.85936780 0.94303690
## cg00121626  0.8774167  0.4066194  0.6659233 0.74479470 0.53358380
## cg00212031  0.7495056  0.3960924  0.1125000 0.13487630 0.08830437
## cg00213748  0.8007889  0.5707763  0.5522088 0.38944720 0.76127050
## cg00214611  0.1002721  0.3796993  0.1153460 0.05165409 0.06238113
##            GSM1183524  GSM1183525 GSM1183526 GSM1183527 GSM1183528
## cg00050873 0.93096740 0.888609500 0.45247250  0.3488372  0.4787234
## cg00063477 0.93944640 0.958587600 0.95414750  0.4899329  0.5841584
## cg00121626 0.74733240 0.602731900 0.28808900  0.3333333  0.3076923
## cg00212031 0.11653450 0.052346570 0.87413560  0.3917749  0.4094293
## cg00213748 0.66512700 0.727069400 0.15711360  0.4723032  0.3415301
## cg00214611 0.05364512 0.009612304 0.02158273  0.5730659  0.5376712
##            GSM1183529 GSM1183530 GSM1183531 GSM1183532 GSM1183533
## cg00050873  0.4054054  0.3913043 0.89683490  0.4303136 0.84521120
## cg00063477  0.3811881  0.5280000 0.92804320  0.5493482 0.93356290
## cg00121626  0.2945946  0.3916667 0.62226690  0.3523810 0.76153420
## cg00212031  0.3295964  0.3172205 0.10158200  0.3865699 0.24342100
## cg00213748  0.4485597  0.2393822 0.65585580         NA 0.67830690
## cg00214611  0.5165165  0.2500000 0.08760108  0.3629764 0.04522613
##            GSM1183534 GSM1183535 GSM1183536 GSM1183537 GSM1183538
## cg00050873  0.3675214  0.4975370  0.2677346  0.4306358  0.4338521
## cg00063477  0.4017341  0.5885827  0.4900990  0.5896415  0.5579598
## cg00121626  0.3304598  0.4157303  0.3324176  0.3623693  0.4563895
## cg00212031  0.3716814  0.3756906  0.3311258  0.4404432  0.2861842
## cg00213748  0.2377359  0.4413265  0.4704545  0.4652778  0.5573334
## cg00214611  0.4838710  0.1891892  0.2355212  0.4900850  0.4490862
##            GSM1183539 GSM1183540 GSM1183541 GSM1183542 GSM1183543
## cg00050873  0.4242424  0.4547619  0.5259117  0.4075630  0.6008287
## cg00063477  0.5742574  0.3647799  0.5036675  0.6674107  0.6009390
## cg00121626         NA  0.5328032         NA  0.3944444  0.4961832
## cg00212031  0.4354839  0.4440433  0.6160221  0.3981132  0.3367609
## cg00213748  0.3104575  0.3088235  0.4068182  0.3901734  0.3036437
## cg00214611  0.3261456  0.2216495  0.2470930  0.4584323  0.4641026
##            GSM1183544 GSM1183545 GSM1183546 GSM1183547 GSM1183548
## cg00050873  0.4068241  0.2845528  0.5178998 0.82567690  0.4810127
## cg00063477  0.4980769  0.4764268  0.6004464 0.95702800  0.5283447
## cg00121626         NA         NA         NA 0.72567680  0.3800905
## cg00212031  0.2788204  0.3000000  0.4541667 0.22278130  0.4071038
## cg00213748  0.3648649  0.2413793  0.1708075 0.78662790  0.2538226
## cg00214611  0.3201581  0.3876923  0.3965517 0.03839441  0.2823530
##            GSM1183549 GSM1183550 GSM1183551 GSM1183552 GSM1183553
## cg00050873  0.3003300 0.82235260  0.6538461 0.89231810  0.5011038
## cg00063477  0.5448852 0.96234980  0.4005682 0.96045950  0.6610577
## cg00121626  0.5718391 0.83983210  0.3573771 0.64023450  0.4781250
## cg00212031  0.4555256 0.03613954  0.3300395 0.03403607  0.3401163
## cg00213748  0.3052959 0.76583710  0.3771044 0.84234530  0.2891986
## cg00214611  0.4277620 0.03083110  0.4011461 0.03042009  0.3382353
##            GSM1183554 GSM1183555 GSM1183556 GSM1183557 GSM1183558
## cg00050873  0.4411086  0.3370288  0.5028681  0.4297297 0.83143660
## cg00063477  0.5256410  0.4371257  0.5286195  0.5035461 0.91331920
## cg00121626  0.3079179  0.4288889  0.3219512  0.2972350 0.82526200
## cg00212031  0.3225000  0.3826291  0.2598039  0.3910761 0.53615280
## cg00213748  0.4161850  0.0661157  0.3219814  0.2637363 0.63163720
## cg00214611  0.2789474  0.2718750  0.4879433  0.4793651 0.04554866
##            GSM1183559 GSM1183560 GSM1183561 GSM1235135 GSM1235136
## cg00050873  0.4301994  0.4411765  0.3226415  0.5119048  0.3230769
## cg00063477  0.4828283  0.5438597  0.3593220  0.4135135  0.4731707
## cg00121626  0.1712707  0.2266667  0.2792553  0.2700000  0.2907268
## cg00212031  0.3475177  0.3658537  0.3230769  0.4090909  0.3373206
## cg00213748  0.4724138  0.4264339  0.4124514  0.4399142  0.2839806
## cg00214611  0.3569482  0.4905213  0.5359116  0.4026667  0.4797136
##            GSM1235137 GSM1235138 GSM1235139 GSM1235140 GSM1235141
## cg00050873  0.1662050  0.3250000 0.74587330  0.3315603 0.76242430
## cg00063477  0.3873626  0.4250559 0.87931690  0.4742857 0.83421050
## cg00121626  0.3544973  0.2643172 0.69287570  0.2906137 0.78437830
## cg00212031  0.1880597  0.3700624 0.60508130  0.4184211 0.62152330
## cg00213748  0.2942708  0.4412533 0.66315790  0.4201183 0.78044160
## cg00214611  0.2887931  0.4444444 0.01397316  0.3958869 0.02109534
##            GSM1235142 GSM1235143 GSM1235144 GSM1235145 GSM1235146
## cg00050873 0.68928370 0.49183260  0.4059406  0.3793814 0.82944610
## cg00063477 0.91036180 0.90426390  0.5597345  0.5504950 0.85077350
## cg00121626 0.86933620 0.73482630  0.4925373  0.2853933 0.47142600
## cg00212031 0.76024150 0.50121790  0.4904679  0.4209184 0.11352250
## cg00213748 0.68796240 0.23240030  0.3102190  0.4420804 0.73678960
## cg00214611 0.02461786 0.02751547  0.4297694  0.3800539 0.06238185
##            GSM1235147 GSM1235148 GSM1235149 GSM1235150 GSM1235151
## cg00050873 0.80426270  0.4379562 0.79032980  0.3483607 0.80677010
## cg00063477 0.86446890  0.3423181 0.88640060  0.4282116 0.93364880
## cg00121626 0.52454530  0.1433225 0.35451100  0.4657534 0.39759150
## cg00212031 0.04820864  0.5205479 0.09348269  0.3240093 0.06174201
## cg00213748 0.38858200  0.4659091 0.81422440  0.4864865 0.67148240
## cg00214611 0.02881844  0.3931889 0.01432064  0.3514986 0.03244385
##            GSM1235152 GSM1235153 GSM1235154 GSM1235155 GSM1235156
## cg00050873  0.3379954  0.6900312  0.5010482 0.57242660  0.2572464
## cg00063477  0.6031469  0.4733728  0.4651741 0.93072930  0.5488889
## cg00121626  0.2500000  0.4373626  0.3675418 0.50813720  0.2905983
## cg00212031  0.3807440  0.3354232  0.2591549 0.56174770  0.1875000
## cg00213748  0.4106383  0.3107143  0.4080268 0.52100840  0.3065015
## cg00214611  0.5197740  0.2871622  0.3147410 0.03402693  0.3260870
##            GSM1235157 GSM1235158
## cg00050873  0.4333333  0.3350785
## cg00063477  0.4590643  0.6019900
## cg00121626  0.4512472  0.2572254
## cg00212031  0.3518072  0.3032787
## cg00213748  0.2485030  0.4775641
## cg00214611  0.2937853  0.3916350
```

```r
head(meta)
```

```
##      group                                title geo_accession
## 1 normal-H Genomic DNA from normal individual 1    GSM1183439
## 2 normal-H Genomic DNA from normal individual 2    GSM1183440
## 3 normal-H Genomic DNA from normal individual 3    GSM1183441
## 4 normal-H Genomic DNA from normal individual 4    GSM1183442
## 5 normal-H Genomic DNA from normal individual 5    GSM1183443
## 6 normal-H Genomic DNA from normal individual 6    GSM1183444
##              tissue colon_region gender stage
## 1 colorectal mucosa        colon   Male      
## 2 colorectal mucosa        colon   Male      
## 3 colorectal mucosa        colon Female      
## 4 colorectal mucosa        colon   Male      
## 5 colorectal mucosa        colon   Male      
## 6 colorectal mucosa        colon   Male
```

For my personal preference I'm gonna reorganize the meta data a bit. 


```r
rownames(meta) <- meta$geo_accession  # move labels to row names
meta$geo_accession <- NULL

meta$title <- as.factor(gsub("Genomic DNA from ", "", meta$title)) #remove redundant words 
head(meta) # yay
```

```
##               group               title            tissue colon_region
## GSM1183439 normal-H normal individual 1 colorectal mucosa        colon
## GSM1183440 normal-H normal individual 2 colorectal mucosa        colon
## GSM1183441 normal-H normal individual 3 colorectal mucosa        colon
## GSM1183442 normal-H normal individual 4 colorectal mucosa        colon
## GSM1183443 normal-H normal individual 5 colorectal mucosa        colon
## GSM1183444 normal-H normal individual 6 colorectal mucosa        colon
##            gender stage
## GSM1183439   Male      
## GSM1183440   Male      
## GSM1183441 Female      
## GSM1183442   Male      
## GSM1183443   Male      
## GSM1183444   Male
```

## Scaling data 

Now let's do clustering. 

But first, as suggested in the seminar tutorial, let's scale so that we look at the distribution of methylation, not the absolute differences of methylation levels among regions. 



```r
s_data <- t(scale(t(data))) 

# for inspection : 
round(data.frame(avgBefore = rowMeans(head(data)),
                 avgAfter = rowMeans(head(s_data)),
                 varBefore = apply(head(data), 1, var),
                 varAfter = apply(head(s_data), 1, var)), 2)
```

```
##            avgBefore avgAfter varBefore varAfter
## cg00050873      0.59        0      0.05        1
## cg00063477      0.66        0      0.05        1
## cg00121626        NA       NA        NA       NA
## cg00212031      0.31        0      0.04        1
## cg00213748        NA       NA        NA       NA
## cg00214611      0.26        0      0.04        1
```

Looks like there are some missing values. I'm gonna scale it again so that this time NA is not being accounted (removing all the rows with NAs), just to make things easier. (good thing we inspect s_data first) 
Note that in this case the probes is much fewer.


```r
nrow(data[complete.cases(data), ]) # 256981 vs 301208 in the original. 
```

```
## [1] 256981
```

```r
s_data_noNA <- t(scale(t(data[complete.cases(data), ])))
```
Maybe doing this would get rid of too many probes. So I'll try replacing NA with zeros in case the NAs in this study mean that they couldn't get any signal (hence zeros)


```r
data_NA_zeros <- data 
data_NA_zeros[is.na(data_NA_zeros)] <- 0
# sanity check
nrow(data_NA_zeros[complete.cases(data_NA_zeros),]) == nrow(data)
```

```
## [1] TRUE
```

Now scale 


```r
s_data_NA_zeros <- t(scale(t(data_NA_zeros)))
```

Just as a reminder: 
- `data` : our original data
- `meta` : our meta data containing what each column in `data` is 
- `s_data`: the scaled `data` , it has NA because `data` has NAs
- `s_data_noNA` : after removing rows with NAs from `data` and then scale it 
- `s_data_NA_zeros` : after replacing NAs with zeros in `data` and then scale it 


## Clustering 


